lib/noid/rails/version.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-module ActiveFedora
-  module Noid
+module Noid
+  module Rails
     VERSION = '2.2.0'
```

Comment:
What do you think of bumping this to 3.0.0.alpha?

Reply:
WFM.